helm/alfresco-content-services/README.md

```diff
@@ -150,8 +150,7 @@ Hence, setting up explicit Container memory and then assigning a percentage of i
 | postgresql-syncservice.image.tag | string | `"13.1.0"` |  |
 | postgresql-syncservice.name | string | `"postgresql-syncservice"` | If true, install the postgresql chart alongside Alfresco Sync service. Note: Set this to false if you use an external database. |
 | postgresql-syncservice.nameOverride | string | `"postgresql-syncservice"` |  |
-| postgresql-syncservice.persistence.existingClaim | string | `"alfresco-volume-claim"` |  |
```

Comment:
Do we really want to remove this from the doc?

Reply:
It's up to helm-docs generation - I wasn't manually changing readmes. Possibly this value is not being anywhere.